hastexo_guacamole_client/consumers.py

```diff
@@ -69,7 +72,11 @@ async def receive(self, text_data=None, bytes_data=None):
         Handle data received in the WebSocket, send to GuacamoleClient.
         """
         if text_data is not None:
-            self.client.send(text_data)
+            # ignore all 'key' events when set to 'read_only" mode
```

Comment:
Just checking: do `key` events include mouse clicks? If not, do we want to disable those as well?

Reply:
Disabled mouse events as well now.